server/conf/cms/settings_default.py

```diff
@@ -1,43 +1,23 @@
-########################
-# DJANGO SETTINGS
-########################
-
 SECRET_KEY = 'replacethiswithareallysecureandcomplexsecretkeystring'
+
 LOGIN_REDIRECT_URL = '/workbench/dashboard/'
 
+SILENCED_SYSTEM_CHECKS = ['captcha.recaptcha_test_key_error']
+
 ########################
-# ELASTICSEARCH
+# SEARCH
 ########################
 
 ES_AUTH = 'username:password'
 ES_HOSTS = 'http://elasticsearch:9200'
 ES_INDEX_PREFIX = 'cep-dev-{}'
 ES_DOMAIN = 'https://cep.test'
 
-es_engine = 'haystack.backends.elasticsearch_backend.ElasticsearchSearchEngine'
 HAYSTACK_CONNECTIONS = {
     'default': {
-        'ENGINE': es_engine,
+        'ENGINE': 'haystack.backends.elasticsearch_backend.ElasticsearchSearchEngine',
         'URL': ES_HOSTS,
         'INDEX_NAME': ES_INDEX_PREFIX.format('cms'),
         'KWARGS': {'http_auth': ES_AUTH}
     }
 }
-
-########################
-# RECAPTCHA SETTINGS
-########################
-
-RECAPTCHA_PUBLIC_KEY = ''
-RECAPTCHA_PRIVATE_KEY = ''
-SILENCED_SYSTEM_CHECKS = ['captcha.recaptcha_test_key_error']
-
-########################
-# REDMINE TRACKER AUTH
-########################
-
-RT_HOST = ''
-RT_UN = ''
-RT_PW = ''
-RT_QUEUE = ''
-RT_TAG = ''
```

Comment:
These were missing from Core-CMS.[^1] I added them just now in https://github.com/TACC/Core-CMS/commit/26bee43.

[^1]: They worked without being added because [they are accessed with a fallback](https://github.com/TACC/Core-CMS/blob/v4.21.0/taccsite_cms/utils/rt.py#L9-L12).